src/components/settings/AppSettingsDialog.vue

```diff
@@ -0,0 +1,45 @@
+<!--
+	SPDX-FileCopyrightText: 2023 Ferdinand Thiessen <rpm@fthiessen.de>
+	SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<template>
+	<NcAppSettingsDialog :open="props.open"
+		:show-navigation="true"
+		name="Logreader settings"
```

Comment:
To match the settings button:
```suggestion
		name="Log reader settings"
```